docs-src/community.md

```diff
@@ -7,14 +7,14 @@ discussions around the APIs.
 
 Feedback and bug reports should be filed as [Github Issues][gh-issues] on this repo.
 
-[gh-issues]: https://github.com/kubernetes-sigs/service-apis/issues/new/choose
+[gh-issues]: https://github.com/kubernetes-sigs/gateway-api/issues/new/choose
```

Comment:
Do the PRs in org and community repo take care of title of Youtube recordings?

Reply:
I think that's still setup relatively manually by an org admin, but I can check to be sure that actually happens.